test/src/sdk-harness/test_projects/tx_fields/mod.rs

```diff
@@ -300,26 +277,39 @@ mod tx {
     #[tokio::test]
     async fn can_get_inputs_count() {
         let (contract_instance, _, wallet, _) = get_contracts().await;
+        let message = &wallet.get_messages().await.unwrap()[0];
+
+        let mut builder = contract_instance.methods()
+            .get_tx_inputs_count()
+            .transaction_builder()
+            .await
+            .unwrap();
+        
+        wallet.adjust_for_fee(&mut builder, 1000).await.unwrap();
 
-        let handler = contract_instance.methods().get_tx_inputs_count();
-        let mut tx = handler.build_tx().await.unwrap();
+        builder.inputs_mut().push(fuels::types::input::Input::ResourceSigned { 
```

Comment:
I would import `fuels::types::input::Input as SdkInput` to avoid full quantifying it all the time.

Reply:
Done.